### ISSUE TITLE
Test Julia Phylonet

### ISSUE THREAD

**Issue by rbartelme** [genophenoenvo/docker#3]
### After Julia integration is complete, test install with mock data from documentation
- [x] Import mock dataset to data store
- [ ] Run mock data through PhyloNetworks

**Comment by rbartelme** [comment]
Moved from sprint 29 to 30.

**Comment by rbartelme** [comment]
Documentation will be created as a separate issue for a future sprint. This method may only be relevant for the Sorghum data.

**Comment by rbartelme** [comment]
I am going to close issue. During testing I uncovered some interoperability issues between the docker invocation of `RUN julia -e 'using Pkg; Pkg.add(["CSV","DataFrames","StatsModels","Gadfly","RCall","PhyloNetworks","PhyloPlots"])'
` vs. what is available in the container.

I tried loading the Julia packages into the Julia lang API from the command line, and nothing loaded. So I think this is a container level issue, and will be creating a new issue to move to the backlog, since this is not a current priority.

New issue is #9
